docs/guides/create_and_upload_dataset.md

```diff
@@ -130,9 +129,11 @@ alice/datasets/mnist/1       3 minutes ago  valid     82.96 MB
 
 Often times, it might not be practical to upload datasets to FloydHub from your local machine. For example, your upload speeds might be too slow, or you just don't want to download a large dataset from the internet just to upload it again.
 
-If your data is already available on the internet, then you can create a dataset directly on FloydHub using Workspaces.
+If your data is already available on the internet, then you can create a dataset directly on FloydHub using Workspaces or from a Job.
```

Comment:
Although it is possible to create dataset from jobs, using workspace is the simpler option for downloading from the internet and tweaking it before creating a new dataset. So I would recommend not adding Job option here for creating new datasets from the internet.